website/docs/guides/cts/text-search.json

```diff
@@ -0,0 +1,166 @@
+{
+  "basic-text-search": {
```

Comment:
`cts run` would run test cases alphabetically, so that would be executed the first – and probably would fail without a text index.

Let's standardize on adding prefixes like `1-` to solve that problem.

Reply:
@noisersup this comment.